src/logion/migration/1637159481048-AddLocFileUniqueConstraint.ts

```diff
@@ -0,0 +1,14 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class AddLocFileUniqueConstraint1637159481048 implements MigrationInterface {
+    name = 'AddLocFileUniqueConstraint1637159481048'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "public"."loc_request_file" ADD CONSTRAINT "UQ_request_id_hash" UNIQUE ("request_id", "hash")`);
```

Comment:
As discussed:

- the primary key should be changed to ("request_id", "hash")
- the unique key should be changed to ("request_id", "index")